c3po/db/dao/link.py

```diff
@@ -16,6 +16,9 @@ class Link(Base):
     post_count = Column("post_count", Integer)
     likes_count = Column("likes_count", Integer)
 
+    yt_views = Column("yt_views", BigInteger)
+    yt_date = Column("yt_date", Date)
```

Comment:
I'm not sure this would be the right way to go. Reasons being:
1. Addition of these fields inherently makes this table to work only with YouTube, which we don't want to do. Even though we support only YouTube today, people share music from multiple platforms (YouTube, Spotify, Apple Music, Gaana, YouTube Music, etc.)
2. The reason we're adding these fields is to calculate popularity and to start with, it is going to be fetched during first share of the song and not update again. So I think it would be much better to dynamically calculate the popularity score and store it in the corresponding field in the `song` table (it should already have this field).